pandas/_libs/tslibs/period.pyx

```diff
@@ -1217,6 +1217,32 @@ cdef class _Period(object):
 
     @property
     def day(self):
+        """
+        Return the number of days of particular month of a given Period.
```

Comment:
This description would make me think this gives the number of days of the month the period falls in, i.e. if the month was February it would return 28. That's obviously note the case - can you think of a way to reword and make clearer?

Reply:
sure I try to make this more clear I'm still thinking that summary is not good sense .
I have query : if we give any period like 2018-03-28 it return 28 so how i explain because I think it count all days till given date ? 
If I think wrong correct me :)

Reply:
I would say "Get day of the month that a Period falls on"

Reply:
I didn't find better sort explanation as you mention on above comment